qcutilities.py

```diff
@@ -1,3 +1,19 @@
+def getAMCFWVersion(shelf, slot, debug=False):
+    """
+    Given an input (shelf,slot) pair, return the FW version of that AMC
+
+    shelf - uTCA shelf idx
+    slot - slot w/in shelf
+    debug - debug flag
+    """
+    
+    from gempython.tools.amc_user_functions_uhal import getAMCObject
+    from gempython.tools.glib_system_info_uhal import getSystemFWVer
+
+    amc = getAMCObject(slot,shelf,debug)
+    
+    return getSystemFWVer(amc,debug)
+
```

Comment:
I don't know that I see the utility of this function.
It looks like you're providing a wrapper around a wrapper around an already existing function?